src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -265,7 +265,7 @@ def do_show(self, args):
             else:
                 gLogger.notice("")
                 gLogger.notice("Setup:", result["Value"]["Setup"])
-                gLogger.notice("DIRAC version:", result["Value"]["DIRAC"])
+                gLogger.notice("DIRAC version:", result["Value"]["Extensions"]["DIRAC"])
                 if result["Value"]["Extensions"]:
```

Comment:
This line is always True, I guess, because DIRAC is always present, should be dropped